packages/create-snap-cli/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@metamask/create-snap",
+  "version": "0.30.0",
```

Comment:
We're on `0.31.1` for the other packages.

Reply:
Should just be `0.31.0`

Reply:
Typo. 😅